src/api/cart.js

```diff
@@ -10,6 +10,12 @@ export default class extends Base {
      */
     getCart(options = {}, callback) {
         let url = '/api/storefront/cart';
+
+        // Cart API requests should always be HTTPS. If the window has an HTTPS url to use, use it.
+        if (window.hasOwnProperty('secureBaseUrl')) {
```

Comment:
I would rather this be passed in as an option, rather than looking for a global variable.